.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
     pip install --upgrade setuptools
     pip install $NUMPY
     pip install $MPL
+    pip install -r requirements.txt
   - |
     cd geos-3.3.3
     export GEOS_DIR=$HOME/.local/
```

Comment:
Right, I suspect we don't need to compile geos anymore. So try taking that
out of the travis yaml. As for shapely, there are definite benefits to
updating the codebase to depend on that very nice package, but I think that
is orthogonal to the problem at hand.

On Tue, Dec 1, 2015 at 1:18 PM, Micah Cochran <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/matplotlib/basemap/pull/234#discussion_r46316025>:
>
> > @@ -32,6 +32,7 @@ install:
> >      pip install --upgrade setuptools
> >      pip install $NUMPY
> >      pip install $MPL
> > +    pip install -r requirements.txt
> >    - |
> >      cd geos-3.3.3
> >      export GEOS_DIR=$HOME/.local/
>
> This is failing because I removed the code from setup.py to compile
> _geoslib.c which is for GEOS library. Oops.
>
> At first glance, shapely <http://toblerity.org/shapely/manual.html> looks
> like it could be replaced GEOS. Also, that would put the burden of
> installing GEOS onto shapely package.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/basemap/pull/234/files#r46316025>.
>